### PR TITLE
NEWS: add some notes for v243

### PR DESCRIPTION
**Pull request by poettering**
Let's get this ball rolling.

### CODE REVIEW

**Comment by poettering** [comment]
(hmm, and to make this easier, let's just merge this quickly, and please post fixes as PRs instead of reviews on this PR)

(@yuwata there was so much stuff in networkd changes, please have a closer look if I missed something!)

File: NEWS [hunk 1]
```diff
@@ -113,6 +117,179 @@ CHANGES WITH 243 in spe:
           overridden on per-service basis. Related setting NUMAMask= is used to
           specify NUMA node mask that should be associated with the selected
           policy.
+
+        * PID 1 will now listen to Out-Of-Memory (OOM) events the kernel
+          generates when processes it manages a reaching their memory limits,
```
Comment (boucman) [inline NEWS:122]: a/are

File: NEWS [hunk 2]
```diff
@@ -113,6 +117,179 @@ CHANGES WITH 243 in spe:
...
+          DHCP lease. It also learnt a new BlackList= option for blacklisting
+          DHCP servers (a similar setting has also been added to the IPv6 RA
+          client), as well as a SendRelease= option for configuring whether to
+          send a DHCP RELEASE message when terminating.
+
+        * systemd-networkd's DHCPv4 and DHCPv6 stacks can now be configured
+          seperately in the [DHCPv4] and [DHCPv6] sections.
+
+        * systemd-networkd's VXLAN support gained a new option
+          GenericProtocolExtension= for enabling XVLAN Generic Protocol
```
Comment (boucman) [inline NEWS:203]: VXLAN ?

File: NEWS [hunk 3]
```diff
@@ -113,6 +117,179 @@ CHANGES WITH 243 in spe:
...
+          ConditionXYZ= expressions.
+
+        * systemd-logind now exposes a per-session SetBrightness() bus call,
+          which may be used to securely change the brightness of a kernel
+          brightness device, if it belongs to the session's seat. By using this
+          call unprivileged clients can make changes to "backlight" and "leds"
+          devices securely with strict requirements on session
+          membership. Desktop environments may use this to generically make
+          brightness changes to such devices without shipping private SUID
+          binaries for that purpose.
```
Comment (boucman) [inline NEWS:262]: .. or specific udev rules (which seems to be the most common way to do it)

File: NEWS [hunk 4]
```diff
@@ -113,6 +117,179 @@ CHANGES WITH 243 in spe:
...
+
+        * systemd-networkd's .network and .link files gained a new Property=
+          setting in the [Match] section, to match against devices with
+          specific udev properties.
+
+        * systemd-networkd's tunnel support gained a new option
+          AssignToLoopback= for selecting whether to use the loopback device
+          "lo" as underlying device.
+
+        * systemd-networkd's MACAddress= setting in the [Network] section has
```
Comment (yuwata) [inline NEWS:241]: [Neighbor] section.

**Comment by yuwata** [comment]
Let's merge this and fix it later, as usual.

**Comment by poettering** [comment]
@boucman please submit such fixes as PR!

**Comment by ssahani** [comment]
When we will release 243 ?

**Comment by poettering** [comment]
@ssahani Well, there are all these that still should be fixed: https://github.com/systemd/systemd/milestones/v243

So it will probably take a bit of time still

**Pull request merged**